blog/02-09-2022-creating-a-provider-for-the-go-sdk.mdx

```diff
@@ -134,17 +134,17 @@ func FetchStoredFlag(key string) (StoredFlag, error) {
 ## Creating a Compliant Provider  
 
 ### Repository Setup  
-`Providers` written for the [`golang-sdk`](https://github.com/open-feature/golang-sdk) are all maintained in the [`golang-sdk-contrib`](https://github.com/open-feature/golang-sdk-contrib.git) repository, containing both `hooks` and `providers`.  
-The following commands can be used to setup the `golang-sdk-conrib` repository, they will clone the repository and set up your provider specific go module under `/providers/MY-NEW-PROVIDER-NAME` adding a `go.mod` and `README.md` file. Your module will them be referenced in the top level `go.work` file.
+`Providers` written for the [`go-sdk`](https://github.com/open-feature/go-sdk) are all maintained in the [`go-sdk-contrib`](https://github.com/open-feature/go-sdk-contrib.git) repository, containing both `hooks` and `providers`.  
+The following commands can be used to setup the `go-sdk-conrib` repository, they will clone the repository and set up your provider specific go module under `/providers/MY-NEW-PROVIDER-NAME` adding a `go.mod` and `README.md` file. Your module will them be referenced in the top level `go.work` file.
 ```
-git clone https://github.com/open-feature/golang-sdk-contrib.git
-cd golang-sdk-contrib
+git clone https://github.com/open-feature/go-sdk-contrib.git
+cd go-sdk-contrib
 make PROVIDER=MY-NEW-PROVIDER-NAME new-provider
 make workspace-init
 ```
 
 ### Creating Your Provider
-In order for your feature flag `Provider` to be compatible with the OpenFeature `golang-sdk` , it will need to comply with the OpenFeature spec. For the `golang-sdk` this means that it will need to fit to the following interface:
+In order for your feature flag `Provider` to be compatible with the OpenFeature `go-sdk` , it will need to comply with the OpenFeature spec. For the `go-sdk` this means that it will need to fit to the following interface:
```

Comment:
```suggestion
In order for your feature flag `Provider` to be compatible with the OpenFeature `go-sdk`, it will need to comply with the OpenFeature spec. For the `go-sdk` this means that it will need to fit to the following interface:
```